packages/dx-react-grid/docs/reference/table-selection.md

```diff
@@ -42,7 +43,7 @@ Field | Type | Description
 selectionAvailable | boolean | True if at least one row can be selected.
 allSelected | boolean | True if all the rows available for selection are selected.
 someSelected | boolean | True if at least one but not all rows available for selection are selected.
-toggleAll | () => void | Selects or deselects all rows.
+toggleAll | (select?) => void | Selects or deselects all rows. If `select` is `true` always select rows, else deselect.
```

Comment:
What will happen if it is called with `undefined` (the optional `select` argument is not specified)?